### ISSUE TITLE
v2: configuration file that informs front-end routes

### ISSUE THREAD

**Issue by ghukill** [WSULib/digitalcollections#18]
We'll need some kind of front-end configuration file that includes things like `APP_HOST`, `LORIS_BASE_PATH`, that kind of thing.

**Comment by colehudson** [comment]
Good point. I think we can store all of those things in the src/settings.php

**Comment by ghukill** [comment]
Good call, that makes sense.
